### PR TITLE
[AAP-9727] Remove /tasks endpoint

### PR DESCRIPTION
**Pull request by Dostonbek1**
* remove /tasks endpoints
* remove /inventory model

JIRA: [AAP-9727](https://issues.redhat.com/browse/AAP-9727)

### CODE REVIEW

**Review by Alex-Izquierdo — COMMENTED** [review]
LGTM, but, what about https://github.com/ansible/eda-server/blob/main/src/aap_eda/api/serializers/project.py#L129 ?

File: src/aap_eda/api/serializers/project.py [hunk 1]
```diff
@@ -126,7 +124,6 @@ def to_representation(self, project):
             "credential": credential,
             "import_state": project["import_state"],
             "import_error": project["import_error"],
-            "import_task_id": project["import_task_id"],
```
Comment (Dostonbek1) [inline src/aap_eda/api/serializers/project.py:129]: @Alex-Izquierdo It is removed here. You might looking at the wrong branch, maybe?

**Review by Alex-Izquierdo — APPROVED** [review]

**Review by hsong-rh — APPROVED** [review]

**Comment by mkanoor** [comment]
@Dostonbek1 @Alex-Izquierdo @hsong-rh There are references to the import_task_id in ansible/ui repo, there is also an openapi.json in that repo we would have to check with @lgalis  on this
The E2E tests are also failing because of this change. https://github.com/ansible/eda-server/actions/runs/7846940638/job/21414768516?pr=657

**Comment by Alex-Izquierdo** [comment]
The main problem right now is the generation of the new openapi spec. @Dostonbek1 is looking it. I understood that he did the double check with @lgalis

**Pull request merged**